src/modules/select/wallets/hyperpay.ts

```diff
@@ -0,0 +1,39 @@
+import { extensionInstallMessage } from '../content'
```

Comment:
```suggestion
import { mobileWalletInstallMessage } from '../content'
```

Since hyperpay is a mobile wallet it will need to display the `mobileWalletInstallMessage`